projects/admin.py

```diff
@@ -125,3 +129,69 @@ class DocumentTemplateAdmin(admin.ModelAdmin):
 @admin.register(ProjectAttributeImage)
 class ProjectAttributeImageAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(PhaseAttributeMatrixStructure)
+class PhaseAttributeMatrixStructureAdmin(admin.ModelAdmin):
+    change_form_template = "admin/matrix.html"
+
+    def render_change_form(
+        self, request, context, add=False, change=False, form_url="", obj=None
+    ):
+        cell_rows = []
+        if obj:
+            x_range = range(len(obj.row_names))
+            y_range = range(len(obj.column_names))
```

Comment:
I think column names is the horizontal, which I think is usually the `x`

Reply:
wups :D

Reply:
Was wondering why I had to reverse the order of the`cell_row` variable I was creating as well 😄